packages/integration/src/lib/workspace/shared/edition-actions.service.ts

```diff
@@ -180,9 +183,46 @@ export class EditionActionsService implements OnDestroy {
         handler: () => {
           this.maximize$.next(false);
         }
+      },
+      {
+        id: 'print',
+        icon: 'printer',
+        title: 'igo.integration.workspace.print.title',
+        tooltip: 'igo.integration.workspace.print.tooltip',
+        handler: (ws: EditionWorkspace) => {
+          const title = `${ws.layer.title} (${getCurrentDateString()})`;
```

Comment:
On pourrait utiliser la transformation de Date proposer par Angular?
Ex: `this.datePipe.transform(Date.now(), "yyyy-MM-dd")`

Reply:
Done.